README.md

```diff
@@ -479,6 +479,7 @@ Twitter 会根据你的喜好推荐其他相关内容，接下来就自由发挥
 
 - [Kubernetes Blog](https://kubernetes.io/blog/)
 - [CNCF Blog](https://www.cncf.io/category/blog/)
+- [CaiCloud Blog](https://caicloud.io/blog)
```

Comment:
Should we use Caicloud or CaiCloud?

Reply:
Caicloud is the right choice.

Reply:
Then

```suggestion
- [Caicloud Blog](https://caicloud.io/blog)
```

Reply:
use official name `Caicloud ` ? 

![image](https://user-images.githubusercontent.com/31201589/61845834-707db900-aed7-11e9-82b5-1e188e1d21d6.png)

Reply:
Yeah, I think so.